Betable/BetableHandlers.h

```diff
@@ -5,8 +5,36 @@
 //  Created by Tony hauber on 7/11/13.
 //  Copyright (c) 2013 Tony hauber. All rights reserved.
 //
+#import <Betable/BetableCredentials.h>
 
 typedef void (^BetableAccessTokenHandler)(NSString *accessToken);
 typedef void (^BetableCompletionHandler)(NSDictionary *data);
 typedef void (^BetableFailureHandler)(NSURLResponse *response, NSString *responseBody, NSError *error);
-typedef void (^BetableCancelHandler)();
\ No newline at end of file
+typedef void (^BetableCancelHandler)();
+typedef void (^BetableLogoutHandler)();
+
+// Game pust provide an implementation of this to betable to facilitate proper game event interactions
+@protocol BetableCredentialCallbacks<NSObject>
+
+@required
+// Betable needs up-to-date insight to current view controller is should game be backgrounded and foregrounded
+- (UIViewController*) currentGameView;
```

Comment:
In practice, its probably the latter.  But I couldn't convince myself that would always be the case--not just usually, especially given Apples perchance for changing the way stuff works--if its not, then lets give the game developer some wiggle room.

As a matter of convenience, I could make the method optional and fallback to the default for UIApplication...